.dockerignore

```diff
@@ -0,0 +1,4 @@
+node_modules
```

Comment:
this example https://github.com/ledermann/docker-vue/blob/master/.dockerignore 
shows additional 
.git
.dockerignore
.gitignore
README.md
Dockerfile

I also usually exclude tests

Reply:
Done